featuretools/entityset/serialize.py

```diff
@@ -2,13 +2,23 @@
 import os
 import shutil
 
-from featuretools import variable_types
+# from featuretools import variable_types
+from featuretools.variable_types import Variable
+
+
+def find_subclasses(cls):
```

Comment:
This is the same code as `_descendants` in `featuretools/primitives/utils.py`.  Let's add this as a function in `featuretools/gen_utils.py` and have both files import it from there